src/main/java/kitchenpos/products/tobe/domain/Product.java

```diff
@@ -0,0 +1,40 @@
+package kitchenpos.products.tobe.domain;
+
+import javax.persistence.*;
+import java.math.BigDecimal;
+
+@Entity
+public class Product {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String name;
+    @Embedded
+    private ProductPrice price;
+
+    public Product() {
```

Comment:
사용하는 곳이 없어 보이네요. public이 아닌 protected로 바꾸면 어떨까요?

```suggestion
    protected Product() {
```